WordPress/src/main/java/org/wordpress/android/push/NotificationsProcessingService.java

```diff
@@ -178,24 +169,22 @@ public void process() {
                 // check special cases for pending draft notifications - ignore
                 if (mActionType.equals(ARG_ACTION_DRAFT_PENDING_IGNORE)) {
                     //dismiss notif
-                    NativeNotificationsUtils.dismissNotification(PENDING_DRAFTS_NOTIFICATION_ID, mContext);
-                    AnalyticsTracker.track(AnalyticsTracker.Stat.NOTIFICATION_PENDING_DRAFTS_IGNORED);
-
-                    // user tapped on ignore, so we need to remember not to notify them again on these drafts
-                    if (mIntent.hasExtra(GROUPED_POST_ID_LIST_EXTRA)) {
-                        ArrayList<Long> groupedPostIdList = (ArrayList<Long>) mIntent.getSerializableExtra(GROUPED_POST_ID_LIST_EXTRA);
-                        AppPrefs.addToPendingDraftsIgnorePostIdList(groupedPostIdList);
-                    } else if (mIntent.hasExtra(POST_ID_EXTRA)) {
-                        long postId = mIntent.getLongExtra(POST_ID_EXTRA, 0);
-                        AppPrefs.addToPendingDraftsIgnorePostIdList(postId);
+                    long postId = mIntent.getLongExtra(NotificationsPendingDraftsReceiver.POST_ID_EXTRA, 0);
+                    if (postId != 0) {
+                        NativeNotificationsUtils.dismissNotification(
+                                NotificationsPendingDraftsReceiver.makePendingDraftNotificationId(postId),
+                                mContext
+                        );
                     }
+                    AnalyticsTracker.track(AnalyticsTracker.Stat.NOTIFICATION_PENDING_DRAFTS_IGNORED);
                     return;
                 }
 
                 // check special cases for pending draft notifications - dismiss
                 if (mActionType.equals(ARG_ACTION_DRAFT_PENDING_DISMISS)) {
                     //dismiss notif
-                    NativeNotificationsUtils.dismissNotification(PENDING_DRAFTS_NOTIFICATION_ID, mContext);
+                    NativeNotificationsUtils.dismissNotification(
+                            NotificationsPendingDraftsReceiver.PENDING_DRAFTS_NOTIFICATION_ID, mContext);
```

Comment:
Should not this be `NotificationsPendingDraftsReceiver.makePendingDraftNotificationId(postId)` ?